tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -32,6 +33,13 @@
 #include "utils.h"
 
 #define DECOMPRESS_CHUNK_CPU_TUPLE_COST 0.01
+
+/* We have to decompress the compressed batches in parallel. Therefore, we need a high
+ * amount of memory. Set the tuple cost for this algorithm a very high value to prevent
+ * that this algorithm is chosen when a lot of batches needs to be merged. For more details,
+ * see the discussion in cost_decompress_batch_merge_append(). */
+#define DECOMPRESS_CHUNK_HEAP_MERGE_CPU_TUPLE_COST 0.8
+
```

Comment:
I guess this should be removed here as it is now in batch_merge.h

Reply:
Thanks, removed the constant.